docs/modules/ROOT/pages/spring-cloud-netflix.adoc

```diff
@@ -131,6 +131,17 @@ public class RestClientConfiguration {
 }
 ----
 
+If you want to control Apache HttpClient 5's protocol specifically, then you can achieve the same
+with the property `eureka.client.hc5.enableProtocolUpgrades`. For example:
+
+.application.yml
+----
+eureka:
+  client:
+    hc5:
```

Comment:
I don't like this abbreviation. Maybe just? `apache`

Reply:
boot uses `httpComponents` in it's `ClientHttpRequestFactoryBuilder`. So `http-components` is another option.

Reply:
The original `eureka.client.hc5` was the result of water cooler brainstorming w/ @OlgaMaciaszek. I've refactored it `eureka.client.http-components`. I am not attached to any of the names, and will be happy to go with what you things gels well with the rest of the ecosystem.

Reply:
We use it this way in OF already, hence my suggestion, but we can move over to `httpComponents` as well.